modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1986,11 +2000,12 @@ public Set<Node> lookupAll(String selector) {
      * Used by Node and Parent for traversing the tree and adding all nodes which
      * match the given selector.
      *
-     * @param selector The Selector. This will never be null.
-     * @param results The results. This will never be null.
+     * @param selector The Selector.
+     * @param results The results.
+     * @return List of matching nodes. The returned value can be null.
```

Comment:
minor: I don't think we should capitalize text in @ param and @ return

Reply:
Agreed. Please change "The" and "List" to lower case.